### DISCUSSION TITLE
@graphql-codegen/typescript-urql for vue

### DISCUSSION THREAD

**Discussion by Manubi** [dotansimha/graphql-code-generator#5806] [Q&A] [answered]
Hi,
is there a way I can add in the config of the /typescript-urql plugin that I am using @urql/vue instead of urql for react?

![CleanShot 2021-04-07 at 15 26 36@2x](https://user-images.githubusercontent.com/20935138/113874134-c088d600-97b5-11eb-93d8-4859c39dd1b1.png)

Another question: If I get the useQuery hook such as:

```
export function useNewsArticlesQuery(
  options: Omit<Urql.UseQueryArgs<NewsArticlesQueryVariables>, "query"> = {}
) {
  return Urql.useQuery<NewsArticlesQuery>({
    query: NewsArticlesDocument,
    ...options
  });
}
```

how would i destructure it in vue? Sorry, I am pretty new to vue.

in reacht I would simply `const [{ data, fetching, error}] = useNewsArticlesQuery();`
Thanks!

**Comment by kamilkisiela** [comment, marked as answer]
I'm not that familiar with Vue but there's `urqlImportFrom` option in [`typescript-urql` plugin](https://www.graphql-code-generator.com/docs/plugins/typescript-urql). You can use it to define your own import `@urql/vue`.

**Reply by Manubi** [reply]
Awesome! thanks!!
